src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -148,43 +149,158 @@ private static int getUserCount(Request req, Response res) {
      * HTTP endpoint to create a "public user" that has no permissions to access projects in the application.
      *
      * Note, this passes a "blank" app_metadata object to the newly created user, so there is no risk of someone
-     * injecting permissions somehow into the create user request.
+     * injecting permissions somehow into the create user request. If the Auth0 user already exists, update the user
+     * permissions instead.
      */
     private static String createPublicUser(Request req, Response res) {
```

Comment:
This endpoint is actually no longer used within the Data Tools application (there is no corresponding UI component), so I'm inclined to take this opportunity to remove it. Apologies for not bringing this up sooner -- after thinking about it for a bit it just doesn't belong in the application any longer.

Reply:
Updated.